splice_machine/src/main/java/com/splicemachine/derby/impl/sql/catalog/upgrade/SpliceCatalogUpgradeScripts.java

```diff
@@ -76,6 +76,8 @@ public SpliceCatalogUpgradeScripts(SpliceDataDictionary sdd, TransactionControll
         this.tc=tc;
 
         scripts = new ArrayList<>();
+        addUpgradeScript(baseVersion4, 1996, new UpgradeConglomerateTable(sdd, tc));
```

Comment:
Please add a comment explaining why this one is out of order.

Reply:
done